File: PRAK604-2310817220016-NailaHanifah.c
#include <stdio.h>
#include <string.h>

int main() {
    char simbol[100], pesan[100];
    int bintang = 0, pagar = 0;

    scanf("%[^\n]%*c %[^\n]%*c", simbol, pesan);
    
    int len = strlen(simbol);

    if (len != strlen(pesan)) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
        return 0;
    }

    for (int i = 0; i < len; i++){
        printf("%c", (simbol[i] == pesan[i]) ? (simbol[i] == ' ' ? ' ' : '*') : '#');
        bintang += (simbol[i] == pesan[i]) && (simbol[i] != ' ');
        pagar += (simbol[i] != pesan[i]);
    }

    printf("\n* = %d\n# = %d\n%s\n", bintang, pagar, bintang >= pagar ? "Pesan Asli": "Pesan Palsu");

    return 0;
}

File: PRAK604-2310817220016-NailaHanifah.py
simbol = input()
pesan = input()

bintang = 0
pagar = 0

if len(simbol) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for i in range(len(simbol)):
        if simbol[i] == pesan[i]:
            print('*', end='' if simbol[i] == ' ' else ' ')
            bintang += 1 if simbol[i] != ' ' else 0
        else:
            print('#', end='')
            pagar += 1

    print("\n* =", bintang)
    print("# =", pagar)
    print("Pesan Asli" if bintang >= pagar else "Pesan Palsu")
